helix-term/src/keymap.rs

```diff
@@ -622,6 +623,8 @@ impl Default for Keymaps {
                 "C-w" | "w" => rotate_view,
                 "C-s" | "s" => hsplit,
                 "C-v" | "v" => vsplit,
+                "f" => goto_file_hsplit,
+                "F" => goto_file_vsplit,
```

Comment:
Looks like you forgot to add these two to the other window map under `space`

Reply:
Ha, added now, thanks!